src/QSymbolicsBase/utils.jl

```diff
@@ -0,0 +1,48 @@
+function prefactorscalings(xs)
+    terms = []
+    coeff = 1::Any
+    for x in xs
+        if isa(x, SScaledOperator)
+            coeff *= x.coeff
+            push!(terms, x.obj)
+        elseif isa(x, Union{Number, Symbolic{Number}})
```

Comment:
In case you were wondering why I added this conditional: when we call `flattenop(*, terms)`, we flatten not only multiplicative products, but scalar products as well. So when this is the case, we need to check if there is a scalar value in any of the terms that `flattenop` gives. If this wasn't here, then `typeof(2*A*B) == SMulOperator` rather than `SScaledOperator`. The latter is what we would want, in my opinion.